torch/_torch_docs.py

```diff
@@ -3836,6 +3836,39 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, min=None, max=None, weight=None, density=False) -> (Tensor, Tensor)
+
+Computes the histogram of the values in a tensor.
+
+Args:
+    {input}
+    bins: int or 1D Tensor. If int, defines the number of equal-width bins. If tensor,
+          defines the sequence of bin edges including the rightmost edge.
+
+Keyword args:
+    min (scalar): Defines the lower range of the bins. If not provided, defaults to input.min().
+    max (scalar): Defines the upper range of the bins. If not provided, defaults to input.max().
+    weight (Tensor): If provided, weight should have the same shape as input. Each value in
+                     input contributes its associated weight towards its bin's result.
+    density (bool): If False, the result will contain the count (or total weight) in each bin.
+                    If True, the result is the value of the probability density function over the bins,
+                    normalized such that the integral over the range of the bins is 1.
+    {out}
```

Comment:
Define this manually. See the second definition of `torch.max` for an example: https://pytorch.org/docs/master/generated/torch.max.html?highlight=max#torch.max.